src/components/GeoMap.js

```diff
@@ -473,7 +472,7 @@ export default class GeoMap {
         } else {
           const originalValueRaw = d.rollupValue[attrKey];
           const originalValue = (attrValue.transform) ?
-              attrValue.transform(originalValueRaw) : originalValueRaw;
+              attrValue.transform(originalValueRaw, this.maptable.rawData) : originalValueRaw;
```

Comment:
yeah...i thought about adding this somewhere. What I'd like to do is actually add an EXAMPLES.md to the parent directory.  Then yanks all the examples of more than a 1 line that break up the README.md and add deep links to EXAMPLES.md.  Sound good?  I'd do this as a separate PR than this one.